### ISSUE TITLE
Jupyterlab won't work with Julia 1.5 and Python 3.7.7

### ISSUE THREAD

**Issue by lewisl** [JuliaLang/IJulia.jl#939]
Note that Jupyter Notebook runs Julia 1.5 properly.

Jupyterlab won't start because my last session, with Julia 1.4.2, had open notebooks.
There is something wrong with the notebooks according to the latest Jupyter. The error appears to come from the startup.jl file that is part of IJulia.

Here are the errors:
```
[I 18:48:15.743 NotebookApp] KernelRestarter: restarting kernel (2/5), new random ports
ERROR: LoadError: ArgumentError: Malformed UUID string: ""
Stacktrace:
 [1] Base.UUID(::SubString{String}) at ./uuid.jl:39
 [2] (::Base.var"#725#726"{String,String})(::IOStream) at ./loading.jl:506
 [3] open(::Base.var"#725#726"{String,String}, ::String; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at ./io.jl:325
 [4] open at ./io.jl:323 [inlined]
 [5] explicit_project_deps_get at ./loading.jl:494 [inlined]
 [6] project_deps_get at ./loading.jl:325 [inlined]
 [7] identify_package(::String) at ./loading.jl:227
 [8] identify_package(::Base.PkgId, ::String) at ./loading.jl:213
 [9] identify_package at ./loading.jl:207 [inlined]
 [10] require(::Module, ::Symbol) at ./loading.jl:888
 [11] include(::Function, ::Module, ::String) at ./Base.jl:380
 [12] include(::Module, ::String) at ./Base.jl:368
 [13] exec_options(::Base.JLOptions) at ./client.jl:296
 [14] _start() at ./client.jl:506
in expression starting at /Users/lewis/.julia/packages/IJulia/DrVMH/src/kernel.jl:1
[I 18:48:18.752 NotebookApp] KernelRestarter: restarting kernel (3/5), new random ports
ERROR: LoadError: ArgumentError: Malformed UUID string: ""
Stacktrace:
 [1] Base.UUID(::SubString{String}) at ./uuid.jl:39
 [2] (::Base.var"#725#726"{String,String})(::IOStream) at ./loading.jl:506
 [3] open(::Base.var"#725#726"{String,String}, ::String; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at ./io.jl:325
 [4] open at ./io.jl:323 [inlined]
 [5] explicit_project_deps_get at ./loading.jl:494 [inlined]
 [6] project_deps_get at ./loading.jl:325 [inlined]
 [7] identify_package(::String) at ./loading.jl:227
 [8] identify_package(::Base.PkgId, ::String) at ./loading.jl:213
 [9] identify_package at ./loading.jl:207 [inlined]
 [10] require(::Module, ::Symbol) at ./loading.jl:888
 [11] include(::Function, ::Module, ::String) at ./Base.jl:380
 [12] include(::Module, ::String) at ./Base.jl:368
 [13] exec_options(::Base.JLOptions) at ./client.jl:296
 [14] _start() at ./client.jl:506
in expression starting at /Users/lewis/.julia/packages/IJulia/DrVMH/src/kernel.jl:1
```

I don't know where this malformed UUID string is.  I did not author this startup.  It comes with the package. This is the kernel definition at a different location.

```
{
  "display_name": "Julia 1.5.0",
  "argv": [
    "/Applications/Julia-1.5.app/Contents/Resources/julia/bin/julia",
    "-i",
    "--startup-file=yes",
    "--color=yes",
    "--project=@.",
    "/Users/lewis/.julia/packages/IJulia/DrVMH/src/kernel.jl",
    "{connection_file}"
  ],
  "language": "julia",
  "env": {},
  "interrupt_mode": "signal"
}
```

**Comment by travigd** [comment]
It's likely that your Project.toml or Manifest.toml file is corrupted.

Try deleting them (or renaming them to something like Project.backup.toml) and trying again.

**Comment by lewisl** [comment]
Thank you.

I did have an incomplete Project.toml file that was the beginning of incomplete package scaffolding.  I did remove it.

However, before I removed it, I deleted all of my Julia jupyter kernels and manually created a new one with installkernel(). This fixed it though I can't really see what is different about the json.

I also started from notebook and navigated to a notebook and opened it. Worked.

I then started directly in Jupyterlab from IJulia.  Worked.

All good. Happy.  Either could have been the culprit. Such simple little things but the breakage the code sees and reports in messages, as we know, often doesn't suggest how to fix to the less knowledgeable user(me). Age old problem that will always be with us.
